File: lecture4/e1.java
import java.util.Scanner;

public class e1{
    public static Scanner scn=new Scanner(System.in);

    public static void main(String[] args)
    {
        System.out.print("Enter size of array: ");
        int n=scn.nextInt();

        int[] arr=new int[n];

        System.out.print("\nEnter values in the array: ");
        for(int i=0;i<n;i++) arr[i]=scn.nextInt();

        display(arr);
    }

    public static void display(int[] arr)
    {
        System.out.print("\nDisplaying the array: ");

        for(int i=0;i<arr.length;i++) System.out.print(arr[i]+" ");
    }
}

File: lecture4/e1.cpp
#include <iostream>
#include <vector>

using namespace std;

void display(vector<int>& arr)
{
    cout<<"\nDisplaying the vector: ";
    for(int i=0;i<arr.size();i++) 
    cout<<arr[i]<<" ";

}

int main(int args,char **argv)
{
    int n;

    cout<<"Enter size of vector: ";
     cin>>n;
    
    vector<int> arr(n,0);

    cout<<"\nEnter the values in vector: ";

    for(int i=0;i<n;i++) cin>>arr[i];

    display(arr);
    return 0;
}
